model_monitoring/components/src/data_drift_compute_metrics/io_utils.py

```diff
@@ -20,7 +20,6 @@ def _get_output_spark_df_schema():
             StructField("metric_value", FloatType(), True),
             StructField("data_type", StringType(), True),
             StructField("metric_name", StringType(), True),
-            StructField("threshold_value", FloatType(), True),
```

Comment:
Is this removing the restriction for threshold_value to be float since in the component code you handle casting nan to float? Or why don't we need this anymore?

Reply:
We're adding the threshold value as a post step after we calculate the metrics. In this case, it assumed it was being given *whilst* calculating the metric which broke some of the algos.

Reply:
We're changing it to be strictly as a post-metric computation step now :)